custom-react-boilerplate/src/components/header/Header.jsx

```diff
@@ -15,18 +16,15 @@ function Header() {
         <Info />
       </div>
       <div className="header-buttons">
-        <Button
-          title="Sign In"
-          className="header-button"
-          onClick={() => {
-          }}
-        />
-        <Button
-          title="Cart"
-          className="header-button"
-          onClick={() => {
-          }}
-        />
+        <Link to="/login" >
```

Comment:
We do not need this link here, instead of links plese use onClick for butten and pass history push.